scripts/maintenance/computational-clusters/autoscaled_monitor/cli.py

```diff
@@ -46,19 +48,41 @@ def _parse_environment(deploy_config: Path) -> dict[str, str | None]:
     return environment
 
 
+def _parse_inventory(deploy_config: Path) -> BastionHost:
+    inventory_path = deploy_config / "ansible" / "inventory.ini"
+    if not inventory_path.exists():
+        rich.print(
+            f"[red]{inventory_path} does not exist! Please run `make inventory` in {deploy_config} to generate it[/red]"
+        )
+        raise typer.Exit(1)
```

Comment:
TIP: there are exit codes defined in built-in `os` see https://docs.python.org/3/library/os.html#os.EX_OK

Reply:
:+1: